### ISSUE TITLE
Breadcrumbs a11y improvements

### ISSUE THREAD

**Issue by pettinarip** [ethereum/ethereum-org-website#5791]
**Description**

Things to improve in the `Breadcrumbs` component:

- Use `nav` as the wrapper element
- Add aria attrs (`aria-label` on the `nav` element)
- Items should not be `h4` should be just `a`
- [Bug] sometimes the last element is empty, leaving an empty `a` with no text. That shouldn’t happen.

**Additional context**

Check as a reference https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by kum9748ar** [comment]
Hi there can I work on this ?

**Comment by pettinarip** [comment]
Thanks @kum9748ar assigned it to you 💪🏼

**Comment by kum9748ar** [comment]
Hi there in breadcrumbs component should I wrap up the `List ` within `nav` like 
![image](https://user-images.githubusercontent.com/67071462/160903266-1c4f1262-9beb-4e42-b2d8-8ab070d2a102.png)
![image](https://user-images.githubusercontent.com/67071462/160903915-2dec3dbb-219a-413a-9f1a-67b34a4fada5.png)

**Comment by kum9748ar** [comment]
@pettinarip Can I get a feedback on my last comment

**Comment by pettinarip** [comment]
@kum9748ar no, we already have a wrapper component. That component is the `List` component.

Now, the `List` component  is currently a `ul` element. You should replace that with a `nav` element.

Take a look at the example in this page https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html we should get a structure similar to that.
```jsx
<nav aria-label="Breadcrumb"> <!-- our List component -->
  <ol>
    <li> <!-- our ListItem component -->
      <a href="../../">
        ...
      </a>
    </li>
  </ol>
</nav>
```

**Comment by pettinarip** [comment]
This one is done.
